modules/oadp-release-notes-1-2-1.adoc

```diff
@@ -27,6 +27,10 @@ For a complete list of all issues resolved in the release of OADP 1.2.1, see the
 
 The following issues have been highlighted as known issues in the release of OADP 1.2.1:
 
+.Backup configuration "csiSnapshotTimeout" not functioning as expected
+
+The backup configuration setting `csiSnapshotTimeout` is not functioning as expected. When attempting to modify this parameter to a custom value, the backup process still utilizes the default timeout value instead of the specified one. This behavior persists regardless of the value assigned to `csiSnapshotTimeout`. link:https://issues.redhat.com/browse/OADP-1952[OADP-1952]
```

Comment:
```suggestion
The backup configuration setting `csiSnapshotTimeout` is not functioning as expected. When attempting to modify this parameter to a custom value, the backup process still utilizes the default timeout value instead of the specified one. This behavior persists regardless of the value assigned to `csiSnapshotTimeout`. link:https://issues.redhat.com/browse/OADP-1952[OADP-1952]
```
```suggestion
The backup configuration setting `csiSnapshotTimeout` is not functioning as expected. When attempting to modify this parameter to a custom value, the backup process still utilizes the default timeout value instead of the specified one. This behavior persists regardless of the value assigned to `csiSnapshotTimeout`. This issue has been resolved in OADP 1.2.2.
```